pkg/actuators/machine/instances.go

```diff
@@ -165,7 +165,7 @@ func getSubnetIDs(machine runtimeclient.ObjectKey, subnet machinev1beta1.AWSReso
 // getAvalabilityZoneFromSubnetID gets an availability zone from specified subnet id.
 func getAvalabilityZoneFromSubnetID(subnetID string, client awsclient.Client) (string, error) {
 	result, err := client.DescribeSubnets(&ec2.DescribeSubnetsInput{
-		DryRun: aws.Bool(true),
+		DryRun: aws.Bool(false),
```

Comment:
What if you just drop this line?

Reply:
I left it like that as there are other similar usages with specifying the `dryRun` as `aws.Bool(false)`.